Fix/return path (#276)
django-ses/django-ses: README.rst

  http://readthedocs.org/docs/boto/en/latest/ref/ses.html#boto.ses.regions
  http://docs.aws.amazon.com/general/latest/gr/rande.html

``AWS_SES_FROM_EMAIL``

>   Optional. The email addres to be used as the "From" address for the email. The address that you specify has to be verified.  

Reviewer: ```suggestion
  Optional. The email address to be used as the "From" address for the email. The address that you specify has to be verified.  
```

>   For more information please refer to http://aws.amazon.com/ses/faqs/#38

Reviewer: Looks like this link needs to be updated

Reviewer: Yes, it doesn't go to any specific question (`#38`)

Reviewer: Yes, seems like a more fitting link! Thank you for suggesting

``AWS_SES_RETURN_PATH``
  Optional. Use `AWS_SES_RETURN_PATH` to receive complaint notifications

>   It only works in the v2 client.

Reviewer: Should this value be used only if `USE_SES_V2` is `True`?

Reviewer 2: Yes, it will only affect emails sent using `USE_SES_V2`.

The V1 WONT'T use this attribute or it's value!

Reviewer: ```suggestion
  You must use the v2 client by setting `USE_SES_V2=True` for this setting to work, otherwise it is ignored.
```

> ``AWS_SES_RETURN_PATH``
>   Optional. Use `AWS_SES_RETURN_PATH` to receive complaint notifications
>   It only works in the v2 client.
>   https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_SendEmail.html#API_SendEmail_RequestSyntax

Reviewer 2: The description of the settings could be the same. This settings was wrongly used.

```suggestion
``AWS_SES_RETURN_PATH``
  Optional. Instruct Amazon SES to forward bounced emails and complaints to this email.
  It only works in the v2 client. v1 client won't use this setting.
  https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_SendEmail.html#API_SendEmail_RequestSyntax
```
